_episodes/03-working-with-data.md

```diff
@@ -25,6 +25,9 @@ OpenRefine displays data in a tabular format. Each row will usually represent a
 
 OpenRefine only displays a limited number of rows of data at one time. You can adjust the number choosing between 5, 10 (the default), 25 and 50 at the top left of the table of data. You can navigate through the records by using the previous/next/first/last navigation options at the top right of the table of data.
 
+OpenRefine comes with a safety belt: note the left panel, currently empty. Read the four words at the top of the panel:
```

Comment:
I'm not sure about the phrasing "OpenRefine comes with a safety belt". I'd suggest that it can more directly say what we mean - In OpenRefine it is always possible to undo any changes - this way we can be clear as possible what we mean.